### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/m1n1/commit/6e02e3cb7e06a3a58b7647189aa886f8406f302e#commitcomment-79066923

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/m1n1/commit/2110bdd110d4bb69364e82035bc3a4d9178b3ba0

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: proxyclient/hv/trace_dcp.py [hunk 1]
```diff
@@ -205,7 +205,7 @@ def handle_cmd(self, hdr, sub, fd):
     "A020": "IOMFB::UPPipeAP_H13P::rt_bandwidth_setup()",
     "A021": "IOMFB::UPPipeAP_H13P::rt_bandwidth_update(IOMFB::AppleRegisterStream*, float, float, bool, bool)",
     "A022": "IOMFB::UPPipeAP_H13P::rt_bandwidth_update_downgrade(IOMFB::AppleRegisterStream*)",
-    "A023": "IOMFB::UPPipeAP_H13P::rt_bandwidth_write_update(IOMFB::AppleRegisterStream*, RealtimeBandwithWritebackBlock, bool)",
+    "A023": "IOMFB::UPPipeAP_H13P::rt_bandwidth_write_update(IOMFB::AppleRegisterStream*, RealtimeBandwidthWritebackBlock, bool)",
```
Comment (jsoref) [inline proxyclient/hv/trace_dcp.py:208]: Neither token appears in google... and the only hit in cs is for this line itself.

File: proxyclient/m1n1/xnutools.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 DebuggerState = Struct(
     "panic_options"             / Hex(Int64ul),
     "current_op"                / Hex(Int32ul),
-    "proceed_on_sync_failre"    / Int32ul,
+    "proceed_on_sync_failure"   / Int32ul,
```
Comment (jsoref) [inline proxyclient/m1n1/xnutools.py:10]: https://opensource.apple.com/source/xnu/xnu-4570.61.1/osfmk/arm/model_dep.c

**Review by jsoref — COMMENTED** [review]
Most corrections were automatically suggested by Google Sheets. All fault is mine.

If certain files come from upstreams, I can try to send patches to them as well (although I have a long list, so there's no particular time guarantee for such activity), and I can certainly easily drop any particular files...

**Comment by andy9a9** [comment]
Isn't possible to squash all commits into one, like style?

**Comment by jsoref** [comment]
codespell only handles _common_ mistakes. check-spelling covers misspellings in general.

I'm happy to squash, but only once people are happy w/ the changes. Often people object to some set of my changes.

File: proxyclient/m1n1/fw/dcp/ipc.py [hunk 3]
```diff
@@ -661,7 +661,7 @@ class IOMobileFramebufferAP(IPCObject):
     D598 = Callback(void, "find_swap_function_gated")
 
 class ServiceRelay(IPCObject):
-    D400 = Callback(void, "get_property", obj=FourCC, key=string(0x40), value=OutPtr(Bytes(0x200)), lenght=InOutPtr(uint))
+    D400 = Callback(void, "get_property", obj=FourCC, key=string(0x40), value=OutPtr(Bytes(0x200)), length=InOutPtr(uint))
```
Comment (jsoref) [inline proxyclient/m1n1/fw/dcp/ipc.py:664]: Argument change

File: proxyclient/hv/trace_dcp.py (hunk 1 above)
Comment (svenpeter42) [reply proxyclient/hv/trace_dcp.py:208]: It's a symbol from apple, it appears in some kext.

**Comment by marcan** [comment]
I think this is fine to merge, but can you fix the conflicts?

File: src/minilzlib/lzmadec.c [hunk 4]
```diff
@@ -291,7 +291,7 @@ LzDecodeLen (
     uint16_t limit;
 
     //
-    // Lenghts of 2 and higher are encoded in 3 possible types of arithmetic-
+    // lengths of 2 and higher are encoded in 3 possible types of arithmetic-
```
[the remark this answers is not in the document]
Comment (jsoref) [reply src/minilzlib/lzmadec.c]: Oops, sorry. I do try to retain case. Dunno how that slipped by. Fixed.

**Comment by marcan** [comment]
Rebased, fixed the conflicts, and squashed. Just one more thing though, can you change the commit author / signoff to a real email instead of the weird github noreply thing? If you tell me what email to use (and consent) I can do it for you too.

Thanks :)

**Comment by jsoref** [comment]
`jsoref@gmail.com`, and sure, go for it.

**Pull request merged**